framework/include/linearfvkernels/LinearFVSource.h

```diff
@@ -33,4 +33,7 @@ class LinearFVSource : public LinearFVElementalKernel
 protected:
   /// The functor for the source density
   const Moose::Functor<Real> & _source_density;
+
+  /// Scale factor
+  const Real & _scale;
```

Comment:
```suggestion
  const Real _scale;
```
for efficiency. If you use a reference, every time the kernel acts it needs to go retrieve the value from the parameters' memory

the only reason to make it a reference would be to make it controllable

Reply:
In FVBodyForce.h it is passed as a reference too. Should this be modified as well?

Reply:
I will apply the suggestion. Just bare in mind there might be other kernels where this is not done this way, e.g. FVBodyForce.h